Guardfile

```diff
@@ -40,7 +40,7 @@ guard :rspec, cmd: ENV['GUARD_RSPEC_CMD'] || 'bundle exec rspec' do
   dsl.watch_spec_files_for(ruby.lib_files)
 
   # Rails files
-  rails = dsl.rails(view_extensions: %w[erb haml slim])
+  rails = dsl.rails(view_extensions: %w[erb haml])
```

Comment:
let's remove `haml` while we're here? Just so nobody gets any fun ideas and tries to sneak one in the future?

Reply:
👍🏼